webpack.config.js

```diff
@@ -51,6 +51,10 @@ module.exports = {
     },
     resolve: {
         extensions: ['.js', '.ts', '.tsx', '.json'],
+        alias: {
+            // Enable absolute imports
+            SRC: path.resolve(__dirname, 'src/ts/')
```

Comment:
in general looks good to me, but I'm curious - why all caps for alias - is that a convention (it bothers me a bit :p)?

Reply:
Oh, I was thinking that the caps communicates to readers that it's an alias, rather than a real path. I agree it's kinda weird.

At my workplace we actually `<PROJECT_ROOT>` for the alias lol

Reply:
I'll go ahead and change it to just `src`